src/nvim/event/multiqueue.h

```diff
@@ -9,7 +9,7 @@
 typedef struct multiqueue MultiQueue;
 typedef void (*PutCallback)(MultiQueue *multiq, void *data);
 
-#define multiqueue_put(q, h, ...) \
+#define MULTIQUEUE_PUT(q, h, ...) \
```

Comment:
I'd disagree with this. it is an uninteresting implementation detail that this is a macro.